doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -107,12 +107,11 @@ application provided procedure also has access to the verify depth information
 and the verify_callback() function, but the way this information is used
 may be different.
 
-SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the limit up
-to which depth certificates in a chain are used during the verification
-procedure. If the certificate chain is longer than allowed, the certificates
-above the limit are ignored. Error messages are generated as if these
-certificates would not be present, most likely a
-X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT_LOCALLY will be issued.
+SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the maximum acceptable
+chain depth to be considered during the verification procedure. Neither
+end-entity nor the trust-anchor certificates count against B<depth>. If the
+certificate chain needed to reach a trusted issuer is longer than B<depth+2>,
+X509_V_ERR_CERT_CHAIN_TOO_LONG will be issued.
 The depth count is "level 0:peer certificate", "level 1: CA certificate",
 "level 2: higher level CA certificate", and so on. Setting the maximum
 depth to 2 allows the levels 0, 1, and 2. The default depth limit is 100,
```

Comment:
Gin the above, setting the depth limit to 2 actually allows levels 0, 1, 2 and 3, with 0 == EE, 1 and 2 are intermediate certificates and 3 == TA.